### ISSUE TITLE
Og-image is empty when running SSG with Qwik

### ISSUE THREAD

**Issue by storrisi** [fabian-hiller/og-img#2]
Probably i'm missing something, but for some reason, when I run a build of a Qwik website, that generates an empty og-image file, even locally.

I'm currently deploying the website over Github Pages, and everytime I make a request towards `website-url/og-image` an empty file is downloaded.

I don't know what should be the correct behavior, probably this won't work with SSG?

Thanks a lot

**Comment by fabian-hiller** [comment]
`og-img` requires a server endpoint. Otherwise, it is not possible to generate a dynamic image, and if your image is static, you probably don't need `og-image` because you could just add an image file to your `/public` directory.

**Comment by fabian-hiller** [comment]
If you want a static site with a dynamic OG image, you could use `og-image` to deploy an AWS Lamda or Google Cloud function independently of your Qwik app. Or you could try Netlify and host everything on the edge. That's what I do with [Valibot](https://valibot.dev/).

**Comment by JoepKockelkorn** [comment]
Ran into this as well, might be good to mention this in the README.

**Comment by fabian-hiller** [comment]
Yes, I will add an info.

**Comment by JoepKockelkorn** [comment]
@fabian-hiller I managed to exclude the route that generates the og-image from SSG (so it uses SSR), but I'm now getting this error:
```
CompileError: WebAssembly.instantiate(): Wasm code generation disallowed by embedder
    at (q-C8RaJ7ou.js:119:7794)
    at (q-C8RaJ7ou.js:119:9924)
    at (q-C8RaJ7ou.js:119:10084)
    at (q-C8RaJ7ou.js:186:29695)
```

I guess this is a security measure?

FYI I've built for Vercel Edge using the proper Qwik City adapter.

Also, this seems equivalent: [someone ran into the same issue but on Cloudflare Pages.](https://github.com/cloudflare/next-on-pages/issues/704)

Edit: [this is the PR I'm testing with](https://github.com/JoepKockelkorn/joepkockelkorn.com/pull/68).
